include/rocksdb/statistics.h

```diff
@@ -539,6 +539,8 @@ class Statistics {
 };
 
 // Create a concrete DBStatistics object
+template <uint32_t TICKER_MAX = TICKER_ENUM_MAX,
+          uint32_t HISTOGRAM_MAX = HISTOGRAM_ENUM_MAX>
```

Comment:
to fix CI, seems we need to keep this API unchanged and make another templated version?

Reply:
it's unrelated, I just forget to include the `.h` for jni